### DISCUSSION TITLE
Feature request: Allow simple conversion of a Pending review comment into an instant single comment

### DISCUSSION THREAD

**Discussion by michcio1234** [community/community#129730] [Pull Requests]
### Select Topic Area

Product Feedback

## Summary
As a PR reviewer, I want to be able to quickly submit my comment as a single comment after I mistakenly clicked "Start a review".

## Background
Countless times I did this:
- review a PR
- find a comment made by another reviewer
- reply to that comment
- mistakenly click "Start a review" instead of "Add single comment".

To fix this, I need to:
- copy the text I wrote (if it contains markup, I first need to hit Edit, then copy, then Cancel)
- delete the comment
- confirm deletion
- paste the text
- click "Add single comment".

This is _a lot_ of work to fix this simple mistake which happens to me all the time.

## Proposed solution
Add a dropdown arrow to the yellow "Pending" label. Once expanded, it reveals an option to "Convert to a single comment". Once clicked, a comment is immediately posted, and is no longer pending.

**Comment by Marenz** [comment]
I want to add that it is _incredibly_ annoying that there is no shortcut to submit a comment without starting a review (CTRL+return starts a review and marks the comment as "pending". IN MY OWN PR EVEN.)
Like, why do I have to go out of my way to respond to a comment on my PR, I hardly want to review and post comments on my own PR as a default action...

**Comment by xmo-odoo** [comment]
This is incredibly frustrating, I just started looking for one such thread after having mistakenly started a review, copied the text back out to post a single comment, done some edition, *and started a review again*. It's incredibly annoying that it takes so much work to revert such a basic error (one which is furthermore encouraged by the github UI, since it creates a review by default and has no shortcut to post single comments).
